ta/os_test/os_test.c

```diff
@@ -406,7 +406,7 @@ static TEE_Result test_mem_access_right(uint32_t param_types,
 	TEE_Result res;
 	uint32_t ret_orig;
 	uint32_t l_pts;
-	TEE_Param l_params[4] = { { {0} } };
+	TEE_Param l_params[TEE_NUM_PARAMS] = { { {0} } };
```

Comment:
Because it's used in TEE_InvokeTACommand
in 
```
res = TEE_InvokeTACommand(sess, 0, TA_OS_TEST_CMD_PARAMS_ACCESS,
				  l_pts, l_params, &ret_orig);
```
And implementation of "RPC" (if I can call it sich in such a way) to another TA uses the same code for handling params in teecore/libutee, as for generic SMC call from NW. 
Without this change xtest os_test will fail.